docs/ruleset_functions/get_zone_peak_internal_load_floor_area.md

```diff
@@ -0,0 +1,49 @@
+# get_zone_peak_internal_load_floor_area
+**Schema Version:** 0.0.21
+**Description:** finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
+
+**Inputs:** 
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **result**: a dict giving 2 values: {"PEAK":total peak btu/h/ft2 in the zone, "AREA":total zone area} the total peak btu/sf is the internal coincident peak loads in all spaces in the zone
+ 
+**Function Call:**
+- **normalize_interior_lighting_schedules**
+
+
+## Logic:
+
+- create list of all loads in the zone.  Each load is an dict of: {"POWER": number, "SCHEDULE": schedule_values}, the peak in btu/h/ft2 takes into account the sensible and latent fraction, for where there are multiple internal_loads in a space, the schedule multiplier indicates the % of total of the internal load: `loads = []`
+- create an area variable which will be the sum of all space areas: `zone_area = 0`
+- For each space in zone: `for space in zone.spaces:`
+	- add the space.floor_area to the area variable: `zone_area += space.floor_area`
+	- create lights power variable for storing the total space lighting power: `lights_power = 0`
+	- For each lighting object in the space: `for lights in space.interior_lighting:`
+		- add the space lighting power to the lights_power & convert from W/m2 to btu/hr: `lights_power += lights.power_per_area * 0.316998331 * space.area`
```

Comment:
I beleive RCT will convert lighting power to W/SF when it parses the RMD, so here we just need to convert W/SF to BTU/hr-sf (i.e., unit conversion should just multiply by 3.41214163

Reply:
@KarenWGard - we've refrained from specifying unit conversions in RDS. @weilixu could you please clariffy what the RCT team expects in such cases.

Reply:
@KarenWGard that doesn't seem to be updated. I think the conversion should be as noted in my first comment above, to go from W/SF to Btu/hr-sf. As Weili mentioned right above, the RCT converts from metric to IP at the time RMD is parsed. So the starting point for this RDS would be lighting expressed as W/ft2, and we need to convert it to BTU/hr/ft2.